### DISCUSSION TITLE
Question about values in the demo

### DISCUSSION THREAD

**Discussion by Nfsaavedra** [cloud-carbon-footprint/cloud-carbon-footprint#1357] [Q&A] [answered]
Hi! I'm trying to manually replicate the calculations of your methodology for a project. However, when I checked the Demo I got confused by the calculation of the MTCO2e from the kWh. In this example, we only have emissions from Azure UK South, which I think it means that the grid emission factors should be [0.000225 metric ton/kWh](https://www.cloudcarbonfootprint.org/docs/methodology/#azure-2). With the energy consumption being 69.14kWh,  I think the carbon footprint should be 0.000225 * 69.14kWh = 0.0155565 MTCO2e. But then, it appears that on the app the value is multipled by 10. Am I doing something wrong with the units of measurement? Thanks!

![image](https://github.com/cloud-carbon-footprint/cloud-carbon-footprint/assets/32967989/b09ed6d0-0025-49ad-94c8-fe72abca534e)

**Comment by ccasher** [comment, marked as answer]
Hi @Nfsaavedra, we try to generally keep our mock data accurate for the demo site, but it should not be held to a degree of reliable accuracy. Your calculations are correct in that the decimal is 1 tenth off. We can try to get this more reliable but we would not suggest using these estimations here as a source of truth!

**Reply by Nfsaavedra** [reply]
Thanks for your quick answer! I understand. Meanwhile, I found that the data is being multiplied by 10 here:
https://github.com/cloud-carbon-footprint/cloud-carbon-footprint/blob/a08e8dbe7e91a4e740fa40347f2a5f49eaad43f3/scripts/update-mock-data.js#L29

**Reply by 4upz** [reply]
Thanks for catching this. This line and the script itself were added to modify the original mock data we were using which were randomized values to a reasonable degree of magnitude since the numbers were too small for demo purposes.  We could remove it, but again want to reiterate that the mock data are just randomly generated kWh values and aren't meant to reflect real calculations. The only point of accuracy we do is convert that randomized energy value into the real co2e equivalent using the appropriate emissions factor for that region (which is also chosen at random)
